editor.planx.uk/src/@planx/components/Content/Public.tsx

```diff
@@ -32,19 +40,81 @@ Content.defaultProps = {
 };
 
 const ContentComponent: React.FC<Props> = (props) => {
+  const {
+    handleSubmit,
+    color,
+    content,
+    info,
+    policyRef,
+    howMeasured,
+    definitionImg,
+  } = props;
+
+  const [open, setOpen] = React.useState(false);
+  const { trackEvent } = useAnalyticsTracking();
+
+  const handleHelpClick = () => {
+    setOpen(true);
+    trackEvent({ event: "helpClick", metadata: {} }); // This returns a promise but we don't need to await for it
+  };
+
   return (
-    <Card handleSubmit={props.handleSubmit} isValid>
+    <Card handleSubmit={handleSubmit} isValid>
       <Content
-        color={props.color}
+        color={color}
         data-testid="content"
-        p={props.color === "#ffffff" || !props.color ? 0 : 2}
+        p={color === "#ffffff" || !color ? 0 : 2}
       >
         <ReactMarkdownOrHtml
-          source={props.content}
+          source={content}
           openLinksOnNewTab
           manuallyIncrementHeaders
         />
       </Content>
+      {!!(info || policyRef || howMeasured) && (
+        <Typography variant="subtitle1" component="div">
+          <HelpButton
+            variant="help"
+            title={`More information`}
+            aria-label={`See more information about this content`}
```

Comment:
This whole block is borrowed straight from our existing `<QuestionHeader />`.

Ideally this `aria-label` would smartly parse out the h1 from the rich text rather than say "this content" but I think this should be sufficient for now?

Reply:
Agreed - I think this is fine for now, and possibly fine full stop?

There's only one context for "this content", and not multiple options (like our `aria-label`s on change buttons for example".

Reply:
One for possible future discussion — should the content component also have a title, separated from the rich text input? It would mitigate the "every page must have a H1" issue we've seen highlighted elsewhere, although a review of the current usage of this component would be needed.

Reply:
We currently have a rich text helper method `incrementHeaderElements()` which automatically adjusts header heirarchy to help combat that "every page must have an H1" issue - it's not perfect, but does help catch a number of users-errors!

Agree that another possible way forward here would be to require more structured input for Content nodes - but then they basically become Notices? Let's keep an eye on it :+1: